src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1139,10 +1140,10 @@ public static bool IsPossibleTupleElementNameContext(this SyntaxToken token, int
 
             if (token.IsKind(SyntaxKind.OpenParenToken))
             {
-                return token.Parent.IsKind(SyntaxKind.ParenthesizedExpression) || token.Parent.IsKind(SyntaxKind.TupleExpression);
+                return token.Parent.IsKind(SyntaxKind.ParenthesizedExpression) || token.Parent.IsKind(SyntaxKind.TupleExpression, SyntaxKind.TupleType);
```

Comment:
Sounds good @jcouv -- I've renamed the method to reflect the more general usage.